tests/next_tests/integration_tests/feature_tests/ffront_tests/ffront_test_utils.py

```diff
@@ -96,7 +96,7 @@ def exec_alloc_descriptor(request):
     for marker, skip_mark, msg in next_tests.definitions.BACKEND_SKIP_TEST_MATRIX.get(
         backend_id, []
     ):
-        if request.node.get_closest_marker(marker):
+        if marker == next_tests.definitions.ALL or request.node.get_closest_marker(marker):
```

Comment:
I would prefer a different approach. I would change line 56 to:
`if dace_iterator and False:`

Reply:
I don't have a strong opinion on this, but you are aware that this means that the tests are not skipped anymore, but simply not executed and hence invisible in the output? No question that this `ALL` thing is ugly, but doing `if dace_iterator and False:` is very hard to find, I honestly completely forgot about this `OPTIONAL_PROCESSORS` altogether.

Reply:
Right, the advantage of the `ALL` marker is visibility. Additionally, it is reusable for other backends.
The reason behind the `OPTIONAL_PROCESSORS` is that dace is an optional dependency for GT4Py, so dace tests are not executed when the dace package is not installed in the python environment.